src/helpers.rs

```diff
@@ -445,6 +445,17 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         'tcx: 'a,
         'mir: 'a,
     {
+        #[cfg(target_os = "unix")]
```

Comment:
That's weird, why does it show this in your diff now even though this is in a PR I landed? Could you rebase and force-push, or so?

Reply:
ugh I messed the git history, I'll try to fix it

Reply:
It's fixed